README.md

```diff
@@ -1 +1,58 @@
-# micro-reddit
+# [Micro-Reddit]
+
+- In this project we worked with data modeling and putting our active record knowledge to practice by building our own light weight Reddit clone.
+
+Models, Views, and Controllers (MVC)
+Data Structures & Relationships
+Routing
+Migrations
+
+ 
+## Setup
+
+- Make a copy in your local enviroment(Clone the Git Repository)
+
+
+
```

Comment:
- **_Include the command to clone the repo_**

- **_Include the commands to setup the project in the local machine (e.g migrate, server...)._**